portfolio/src/main/webapp/index.html

```diff
@@ -22,19 +22,26 @@ <h1><a href="index.html" class="header-text">Iris Xu</a></h1>
       </div>
     </nav>
 
-    <!-- TODO: About section -->
-    <div id="content">
-      <h1>Iris' Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-
-    <!-- TODO: Projects section -->
-
-    <!-- TODO: Personal section -->
+    <main class="container">
+      <!-- TODO: About section -->
+      <section id="hero">
+        <h1>Hello, my name is Iris.</h1>
+      </section>
+
+      <div id="content">
+        <h1>Iris' Portfolio</h1>
+        <p>This is my portfolio.</p>
+        <p>Click <a href="/hello">here</a> to see content created by server code.</p>
```

Comment:
This week we're focusing on client side development and next week will be server-side. Are you adding some server code for this week?

Reply:
This part is being deleted.